cmd/zed/lake/flags.go

```diff
@@ -48,6 +48,9 @@ func (f *Flags) Open(ctx context.Context, engine storage.Engine) (*lake.Root, er
 	if err != nil {
 		return nil, err
 	}
+	if root.Path == "" {
+		return nil, errors.New("no lake path specied: use -R or ZED_LAKE_ROOT environment")
```

Comment:
Nit The phrase "use -R or ZED_LAKE_ROOT environment" reads oddly. How about "use -R or ZED_LAKE_ROOT environment variable" or "use -R or set ZED_LAKE_ROOT"?